jdk_8_maven/em/embedded/rest/ocvn/src/main/resources/init_db.sql

```diff
@@ -34,6 +34,35 @@ VALUES
      NULL
 );
 
+INSERT INTO PERSON
+(
+    CHANGE_PASSWORD,
+    COUNTRY,
+    EMAIL,
+    ENABLED,
+    FIRST_NAME,
+    LAST_NAME,
+    PASSWORD,
+    TITLE,
+    USERNAME,
+    GROUP_ID
+)
+VALUES
+    (
+     false,
+     NULL,
+     NULL,
+     true,
+     NULL,
+     NULL,
+     'b65d4278fb536a6e1ea23c87988f5d79ef743ccb07b2067ad8ebd52f1a0c067e5bc3b7f76eca3abc',
+     NULL,
+     'user1',
+     NULL
+);
+
 INSERT INTO PERSON_ROLES(PERSON_ID,ROLES_ID)VALUES(1,2);
 INSERT INTO PERSON_ROLES(PERSON_ID,ROLES_ID)VALUES(1,1);
 INSERT INTO PERSON_ROLES(PERSON_ID,ROLES_ID)VALUES(1,3);
+
+INSERT INTO PERSON_ROLES(PERSON_ID,ROLES_ID)VALUES(2,1);
```

Comment:
shouldn't `jdk_8_maven/em/external/rest/ocvn/src/main/resources/init_db.sql` be modified as well?

Reply:
Hi Andrea, thanks for your comment. I added those to both init_db.sql scripts (for embedded and external).